clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/subsystem/StateTransferResourceDefinition.java

```diff
@@ -55,13 +57,20 @@
     static final PathElement PATH = pathElement("state-transfer");
 
     enum Attribute implements org.jboss.as.clustering.controller.Attribute {
+        ENABLED("enabled", ModelType.BOOLEAN, new ModelNode(true)),
         CHUNK_SIZE("chunk-size", ModelType.INT, new ModelNode(512)),
         TIMEOUT("timeout", ModelType.LONG, new ModelNode(TimeUnit.MINUTES.toMillis(4))),
         ;
         private final AttributeDefinition definition;
 
         Attribute(String name, ModelType type, ModelNode defaultValue) {
-            this.definition = createBuilder(name, type, defaultValue).build();
+            this.definition = new SimpleAttributeDefinitionBuilder(name, type)
+                    .setAllowExpression(true)
+                    .setRequired(false)
+                    .setDefaultValue(defaultValue)
+                    .setMeasurementUnit((type == ModelType.LONG) ? MeasurementUnit.MILLISECONDS : null)
```

Comment:
According to its description, `chunk-size` should have a `BYTES` mesaurement unit.

Reply:
chunk-size refers to the number of cache entries.  I've opened https://issues.jboss.org/browse/WFLY-9764 to fix the description.